pkg/ngctl/cmd/util/factory.go

```diff
@@ -36,9 +36,9 @@ type (
 		ToRuntimeClient() (client.Client, error)
 		GetNebulaClusterNameAndNamespace(withUseConfig bool, args []string) (string, string, error)
 		GetNebulaClusterNamesAndNamespace(withUseConfig bool, args []string) ([]string, string, error)
-		// GetNebulaClusterName() (string, error)
+		GetNebulaClusterName() (string, error)
 		// GetNebulaClusterNameWithoutConfig() string
-		// GetNamespace() (string, error)
+		GetNamespace() (string, error)
```

Comment:
`GetNebulaClusterName` and `GetNamespace` may cause out of sync, can you use `GetNebulaClusterNameAndNamespace` instead?